resources/views/Public/ViewEvent/Embedded/EventPageCheckout.blade.php

```diff
@@ -1,7 +1,7 @@
 @extends('Public.ViewEvent.Layouts.EmbeddedEventPage')
 
 @section('head')
-<script type="text/javascript" src="https://js.stripe.com/v2/"></script>
+<!-- script type="text/javascript" src="https://js.stripe.com/v2/"></script -->
```

Comment:
Oh, yeah, I didn't intend to disable that; I just debugged it and temporarily commented it--will put it back ;-)